decidim-admin/app/assets/javascripts/decidim/admin/application.js.es6

```diff
@@ -35,6 +35,17 @@ const pageLoad = () => {
       );
     }
   })
+
+  const $participatoryProcessScopeEnabled = $('#participatory_process_scope_enabled');
+  const $participatoryProcessScopeId = $("#participatory_process_scope_id");
+
+  if ($('.edit_participatory_process').length > 0) {
+    $participatoryProcessScopeEnabled.on('change', (event) => {
+      const checked = event.target.checked;
+      $participatoryProcessScopeId.attr("disabled", !checked);
+    })
+    $participatoryProcessScopeId.attr("disabled", !$participatoryProcessScopeEnabled.prop('checked'));
+  }
```

Comment:
Can we add this in a `participatory_processes.js.es6` file? That way the functionality can be easily duplicated for other participatory spaces. I guess the toggle could be made independent from participatory processes directly, but just extracting it to a separate file would be good enough for me.